src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw.module.ts

```diff
@@ -83,8 +83,9 @@ import {
   TooltipModule,
   ComboBoxModule,
   ToggletipModule,
+  IconService,
   LayoutModule,
-  IconService
```

Comment:
This does not seem right

Reply:
This seems to be unlreated to the PR and happened becuase on tentacle, `IconService` was below `LayoutModule` while on main was the opposite. Now matches main branch so this is fine.